### PR TITLE
Bug/2524/invalid answer fields

### PR DESCRIPTION
**Pull request by njaeggi**
fixes: #2524

### CODE REVIEW

**Comment by njaeggi** [comment]
Things to note for reviewer:
- Make sure the the Person used for testing can't edit the Event
- Check Radio Buttons, Checkboxes and normal text fields

File: app/helpers/standard_form_builder.rb [hunk 1]
```diff
@@ -212,10 +212,13 @@ def datetime_field(attr, html_options = {})
 
   def inline_radio_button(attr, value, caption, inline = true, html_options = {})
     html_options[:class] = html_options[:class].to_s
-    html_options[:class] += ' is-invalid' if errors_on?(attr)
+    html_options[:class] += ' form-check-input allign-label'
```
Comment (kronn) [inline app/helpers/standard_form_builder.rb:215]:
```suggestion
    html_options[:class] += ' form-check-input align-label'
```

File: app/helpers/standard_form_builder.rb [hunk 2]
```diff
@@ -230,11 +233,13 @@ def inline_nested_form_custom_checkbox(attr, value, index)
...
     hidden_field = index.zero? ? @template.hidden_field_tag(name, index) : ''
+    invalid = errors_on?(attr) ? " is-invalid" : ""
 
-    @template.label_tag(sanitized_id, class: 'checkbox') do
-      hidden_field.html_safe +
-      @template.check_box_tag(name, index + 1, checked, id: sanitized_id) +
-      ' ' +
+    @template.check_box_tag(name, index + 1, checked, 
+                            id: sanitized_id, 
+                            class: "form-check-input allign-label" + invalid) +
```
Comment (kronn) [inline app/helpers/standard_form_builder.rb:240]:
```suggestion
                            class: "form-check-input align-label" + invalid) +
```

File: app/javascript/stylesheets/hitobito/_form.scss [hunk 3]
```diff
@@ -857,3 +857,8 @@ fieldset > div > div.labeled {
 .mw-63ch {
   max-width: 63ch;
 }
+
+.allign-label {
```
Comment (kronn) [inline app/javascript/stylesheets/hitobito/_form.scss:861]:
```suggestion
.align-label {
```

**Review by kronn — CHANGES_REQUESTED** [review]
ausrichten: to align

Ansonsten cool und mergebar.

**Review by kronn — APPROVED** [review]

**Pull request merged**